packages/portfolio-contract/test/portfolio.flows.test.ts

```diff
@@ -513,6 +521,53 @@ test('open portfolio with Aave position', async t => {
   await documentStorageSchema(t, storage, docOpts);
 });
 
+test('open portfolio and claim with Aave position', async t => {
+  const { add } = AmountMath;
+  const amount = AmountMath.make(USDC, 300n);
+  const feeAcct = AmountMath.make(BLD, 50n);
+  const feeCall = AmountMath.make(BLD, 100n);
+  const { orch, tapPK, ctx, offer, storage } = mocks(
+    {},
+    { Deposit: amount, GmpFee: add(feeAcct, feeCall) },
```

Comment:
2 steps below use `feeCall`. So unless we combine them, I think that should be:


```suggestion
    { Deposit: amount, GmpFee: add(feeAcct, add(feeCall, feeCall)) },
```